pkg/client/cache/store.go

```diff
@@ -164,6 +164,11 @@ func (c *cache) Index(indexName string, obj interface{}) ([]interface{}, error)
 	return c.cacheStorage.Index(indexName, obj)
 }
 
+// ListIndexNames returns a list of names used by the Index
+func (c *cache) ListIndexNames(name string) []string {
+	return c.cacheStorage.ListIndexNames(name)
+}
+
```

Comment:
@derekwaynecarr would you mind reviewing this change?  I added the ability to retrieve index names from an Index.